pages/interface/components/ReadTime/index.js

```diff
@@ -0,0 +1,16 @@
+import { Text } from '@/TabNewsUI';
+import { useMemo } from 'react';
+
+export default function ReadTime({ text, ...props }) {
+  const readTimeInMinutes = useMemo(() => {
+    const wpm = 260;
+    const wordsQuantity = text.split(' ').length;
```

Comment:
```suggestion
    const wordsQuantity = text.split(/[^A-Za-z]+/).length;
```

Uma alternativa um pouco mais precisa quando houver sequências de espaços, quebras de linhas e outros caracteres diferentes de letras.

Reply:
feito!